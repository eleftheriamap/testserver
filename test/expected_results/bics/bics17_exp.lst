
test/exp/test/gen/bics/bics17_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	911f814a 	add	x10, x10, #0x7e0
   4:	d144bab5 	sub	x21, x21, #0x12e, lsl #12
   8:	ea356d49 	bics	x9, x10, x21, lsl #27
   c:	8a000000 	and	x0, x0, x0
