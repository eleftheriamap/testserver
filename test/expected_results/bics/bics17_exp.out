Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 00000000000007e0
X10    = 00000000000007e0
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = ffffffffffed2000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : ----
Non-zero memory:
0x00000000 : 0x911f814a
0x00000004 : 0xd144bab5
0x00000008 : 0xea356d49
0x0000000c : 0x8a000000
